hack/go-lint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/env bash
+#!/bin/bash
```

Comment:
This was change some time ago to `#!/bin/env bash`
@ricardozanini can you recall why ?

Reply:
Ubuntu local clients.